### DISCUSSION TITLE
[Feature] <Photos assigned to people not hidden when archiving>

### DISCUSSION THREAD

**Discussion by mnisyif** [immich-app/immich#11086] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I do have some photos that i like to hide of my wife, and generally that way i do it is by adding the photos to an album and archive the photos. The problem is that it is also hidden from the photos assigned to a person in the `People` tab. 

It would be nice if the photos stay consistent in `People`'s tab, and hidden from the timeline.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bkbilly** [comment]
I am also archiving the images when I add them to a photo album so that I could have a cleaner timeline.
I agree with @mnisyif that the people's tab should show the archived images.

**Comment by bo0tzz** [comment]
On my end archived images are visible in the people tab

**Reply by bkbilly** [reply]
I've archived the images of 1 person to test it and it doesn't show at people tab and the only way to check his images is by going to the archived image and click on it. Even though it has about a dozen images. it shows that it has 0 assets. 
You can see this behavior on the following image:

![Screenshot from 2024-09-08 11-32-46](https://github.com/user-attachments/assets/ff4d9894-6da6-4445-b94a-5bfff7ac3b45)
